qa/tasks/cephfs/fuse_mount.py

```diff
@@ -258,8 +258,14 @@ def wait_until_mounted(self):
     def _mountpoint_exists(self):
         return self.client_remote.run(args=["ls", "-d", self.mountpoint], check_status=False, cwd=self.test_dir, timeout=(15*60)).exitstatus == 0
 
-    def umount(self):
+    def umount(self, cleanup=True):
+        """
+        umount() must not run cleanup() when it's called by umount_wait()
+        since "run.wait([self.fuse_daemon], timeout)" would hang otherwise.
+        """
```

Comment:
Yeah, this change makes sense, but I couldn't see why ?

Reply:
Here's why - https://github.com/ceph/ceph/pull/34951#discussion_r424385618.